script/ghIssues.pl

```diff
@@ -140,12 +140,15 @@ sub getIssues{
                 } )->hri->one_row;
                 my $new_ia = 1 if !$ia;
 
+                # no auto generating IA pages from a PR anymore
+                return unless $ia;
+
                 my @time = localtime(time);
-                my $date = "$time[4]/$time[3]/".($time[5]+1900);
+                my ($month, $day, $year) = ($time[4]+1, $time[3], $time[5]+1900);
```

Comment:
Pages to test in order to see whether the dates are parsed correctly:
- Live IA Page with issues (eg ia/view/calculator)
- Overview (ia/dev)
- Live Issues (ia/dev/issues)
- Dev Pipeline (ia/dev/pipeline)